frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM cypress/base:latest
+
+WORKDIR /app 
+
+COPY . /app 
+
+EXPOSE 3000
+
+RUN npm install
```

Comment:
What is this Dockerfile used for?
I think the container will go down at the end of the installation.

Reply:
It's used by the docker-compose to build. 

Sorry I mixed up RUN and CMD. Container will go down at the end of CMD. The command is in the docker-compose so everything is fine here.